jan5_assign.html

```diff
@@ -1,32 +1,37 @@
 <script>
   var question, 
-      answer,
-      guess,
-      count,
-      pass;
-    
-  question = ("Can you guess the number of characters that make up this question?");
+    answer,
+    guess,
+    count,
+    pass;
+  
   alert("You have a maximum of 3 attempts to get this right");
+  question = ("Can you guess the number of characters that make up this question?");
+  
   guess = prompt(question);
   answer = question.length;
-
   count = 1;
-
-  while(count < 3){
+  pass = false;
+  
+  while((count < 3) && (pass == false)) {
```

Comment:
because pass has a boolean value, you don't have to compare it to false.
`pass == false` is equivalent to `!pass`
so this line could be 
`while((count < 3) && !pass) {`